sdks/rust/.gitignore

```diff
@@ -13,5 +13,9 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-
 # End of https://www.gitignore.io/api/rust
+
+# The protocol buffers need to be adjacent to the cargo manifest when publishing
+# due to cargo protections, but for local iteration (without going through images)
+# we don't want to accidentally check them in
+/proto/
```

Comment:
🤔 do we actually want to ignore these? I don't mind that they are checked in. I mean, if people want to use the folder as a library, they would have to be there for the end user - yes? Otherwise build.rs is going to fail because the protos aren't there.

We also wouldn't be able to publish this as a crate. So I don't think we need to ignore them. We'll just have them vendored in two places, with the gen.sh script to automate keeping the rust files up to date. Since the other gen.sh actually generates vendored code, I think that's acceptable. So I think we can remove this ignore.